tests/integration/pilot/common/routing.go

```diff
@@ -129,7 +133,7 @@ metadata:
   name: gateway
 spec:
   selector:
-    istio: ingressgateway
+    istio: {{.GatewayIstioLabel | default "ingressgateway"}}
```

Comment:
nit: seems we already do default on L87

Reply:
Turns out, L87 was completely unnecessary and the result of some bad git patching. There's no defaulting on L154, so L136 is necessary.